### DISCUSSION TITLE
Tag generation running through Pipeline

### DISCUSSION THREAD

**Discussion by milafrerichs** [open-webui/open-webui#6817] [Q&A]
I have a pipeline and was always wondering where the extra requests were coming from until I finally saw it. 
Once the chat is completed the App call generateTags and this call the /api/task/tags/completions endpoint which in turn creates a new request to the model and therefore the RAG. 
is this the intended purpose? Or how can I prevent that? 
Because it is searching my vector database for the generate tags prompt. 

thanks.

**Comment by nikoizs** [comment]
Hi @milafrerichs , you can get around it by setting the "External Task Model" to something else than "Current Model"
When it is set to "Current model" it will use your pipeline to generate tags and titles .

it's under Settings -> Interface -> Set Task Model -> External Model
